ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
@vijayendrabvs @mattklein123 The push could be done after the ubuntu based push here https://github.com/lyft/envoy/blob/master/ci/docker_push.sh#L7, I don't see why we couldn't have two images pushed. 

The only problem I see is that at that point we have an unstripped version of the binary, so you'd have to strip it in your image. Also the path in your Makefile and Dockerfile is not where the binary would be at that point in CI, so you would need to fix that.